jekyll/_cci2/server/v4.4/overview/circleci-server-overview.adoc

```diff
@@ -19,11 +19,17 @@ CircleCI server is an on-premises CI/CD platform for enterprise customers who ha
 
 CircleCI server provides the same features as CircleCI’s cloud offering, but operates within your Kubernetes cluster.
 
-NOTE: For installation instructions, you have the option of either the xref:../installation/phase-1-prerequisites#[regular] or xref:../air-gapped-installation/phase-1-prerequisites#[air-gapped] installation guide.
+[#architecture]
+== Architecture
+
+The following diagram presents and overview of the services architecture of CircleCI server.
 
 .CircleCI server v4.4 Architecture
 image::server-4-architecture-diagram.png[Server v4.4 services architecture]
 
+[#ports-and-services]
+=== Ports and services
+
 The CircleCI server application exposes four services, using a single load balancer. If required, the load balancer can be made private, separating it from the public internet.
```

Comment:
Is this still correct? Four?

Reply:
Ah good catch. This is now just two services:
* nginx
* nomad-server

Reply:
👍